projectLive/R/mod_about_page.R

```diff
@@ -68,72 +42,77 @@ mod_about_page_ui <- function(id){
 #' @rdname mod_about_page
 #' @export
 #' @keywords internal
-    
+
+
 mod_about_page_server <- function(input, output, session){
   ns <- session$ns
   
+  output$about <- shinydashboard::renderInfoBox({
+    
+    shinydashboard::infoBox(
+      " ",
+      print("projectLive: Track the progress and impact of our funding partners in real time"),
+      icon = icon("university", "fa-1x"),
+      color = "light-blue", #Valid colors are: red, yellow, aqua, blue, light-blue, green, navy, teal, olive, lime, orange, fuchsia, purple, maroon, black.
+      fill = TRUE
+    )
+  })
+  
+  
+  # Add synapse function
+  syn <- synapseclient$Synapse()
+  
+  # Adding in-app authentication
+  session$sendCustomMessage(type = "readCookie", message = list())
+  
+  # Show message if user is not logged in to synapse
+  unauthorized <- shiny::observeEvent(input$authorized, {
+    shiny::showModal(
+      shiny::modalDialog(
+        title = "Not logged in",
+        HTML("If you are a member of the funding team, you must log in to <a target=\"_blank\" href=\"https://www.synapse.org/\">Synapse</a> to access this resource. Please log in, and then refresh this page.") # nolint
+      )
+    )
+  })
+  
+  
+  # If logged in, store userid to check access
+  #observeEvent(input$cookie, {
```

Comment:
Line 80 will be uncommented in final version. For now the functionality is being tested using syn$login()

Reply:
addressed in #49